lib/connection-pool.js

```diff
@@ -77,7 +80,7 @@ function createConnection(pooled) {
 
         this.connections.push(pooled);
```

Comment:
the event gives the ability to get notified before AND after a connection has been put into the "connections" list